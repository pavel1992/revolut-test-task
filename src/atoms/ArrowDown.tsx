import styled from "styled-components";

import { colors } from "../colors";

export const ArrowDown = styled.div`
  position: absolute;
  left: calc(50% - 25px);
  bottom: -25px;
  width: 0;
  height: 0;
  border-style: solid;
  border-width: 25px 25px 0 25px;
  border-color: ${colors.thinBorderWhite} transparent transparent transparent;
  z-index: 1;
`;
