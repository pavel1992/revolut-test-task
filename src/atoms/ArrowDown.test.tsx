import * as React from 'react';
import renderer from 'react-test-renderer';

import { ArrowDown } from "./ArrowDown";


test('renders arrow correctly', () => {
  const render = renderer
    .create(<ArrowDown/>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
