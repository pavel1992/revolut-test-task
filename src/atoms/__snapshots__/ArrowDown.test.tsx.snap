// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders arrow correctly 1`] = `
<div
  className="sc-AxjAm FESAN"
/>
`;
